keys.go

```diff
@@ -355,6 +356,10 @@ func (c *Client) GetPolicy(ctx context.Context, id string) (*Policy, error) {
 		return nil, err
 	}
 
+	if policyresponse.Policies == nil {
+		return nil, errors.New("No policies")
```

Comment:
Should we return an error or just return `nil`? I don't think its an error if there is no policy.

We should add this case to the function doc string as well.
```// GetPolicy retrieves a policy for a given Key ID. If there is no policies for the key, returns nil.```